lang/cs/Org.Apache.REEF.Client/YARN/IYarnCommandLineEnvironment.cs

```diff
@@ -30,6 +30,6 @@ internal interface IYarnCommandLineEnvironment
         /// Returns the class path returned by `yarn classpath`.
         /// </summary>
         /// <returns>The class path returned by `yarn classpath`.</returns>
-        IList<string> GetYarnClasspathList();
+        IReadOnlyList<string> GetYarnClasspathList();
```

Comment:
@markusweimer Yes, this is indeed is a breaking change. I recently added this interface (within this release cycle). I would much rather revert this change than go through `Obsolete` song and dance. 
This being mutable gave me a lot of grief in debugging a test failure and is definitely a possible source of error. Let me know what you want.